src/components/pages/login/TitleStyled.js

```diff
@@ -0,0 +1,34 @@
+import styled from "styled-components"
+import logo from "../../../images/logo-orange.png"
+import { theme } from "../../../theme"
+
+function MainTitle() {
+   return (
+      <TitleContainer>
+         CRAZEE
+         <img src={logo} alt="logo crazy burger" />
```

Comment:
```suggestion
         <img src={logo} alt="logo-crazy-burger" />
```

(par convention de naming du alt)